index.html

```diff
@@ -0,0 +1,132 @@
+        <div><hr><li class="nav-bar__item"><a href="#">Sponsors</a></li></div>
+        <div><hr><li class="nav-bar__item"><a href="#">News</a></li></div>
+        <div><hr><li class="nav-bar__item"><a class="nav-bar__item--design" href="#">NE Campaign</a></li><hr></div>
+      </ul>
+    </nav>
+  </header>
+
+  <section class="main">
+    <div class="container">
+      <div class="main__button"><button><img class="w-50" src="icons/nav_bar_icon.png" alt="Open"></button></div>
+      <div class="main__title">
+        <h2>"N'AI'tion Evolution"</h2>
+        <h1>NAITION EVOLUTION GLOBAL CONFERNECE 2024</h1>
+      </div>
+      <div class="main__desc">
+        <p>An extense conference talking about AI advances and tech. This conference is taking place in July, in California.</p>
+      </div>
+      <div class="main__date">
+        <h2>2024.07.19(FRI) ~ 20(SAT)</h2>
+        <h3>@ Event Spaces - Guerneville, CA</h3>
+      </div>
+    </div>
+  </section>
+
+  <section id="program" class="program">
+    <div class="container">
+      <div class="title-bar"><h2>Main Program</h2><hr></div>
+      <div class="program__cont">
+        <div class="program__card flex">
+          <div class="program__card--icon"><img src="icons/program_icon_01.png" alt="Icon"></div>
+          <div class="program__card--title"><h2>Intro</h2></div>
+          <div class="program__card--text"><p>A little introduction that explains the objective and important points of the conference.</p></div>
+        </div>
+        <div class="program__card flex">
+          <div class="program__card--icon"><img src="icons/program_icon_02.png" alt="Icon"></div>
+          <div class="program__card--title"><h2>Exhibition</h2></div>
+          <div class="program__card--text"><p>Appreciate various creations applying NE license of artists, organized from Art Center.</p></div>
+        </div>
+        <div class="program__card flex">
+          <div class="program__card--icon"><img src="icons/program_icon_03.png" alt="Icon"></div>
+          <div class="program__card--title"><h2>Forum</h2></div>
+          <div class="program__card--text"><p>Have the time to share your thoughts and opinions with experts for each topic.</p></div>
+        </div>
+        <div class="program__card flex">
+          <div class="program__card--icon"><img src="icons/program_icon_04.png" alt="Icon"></div>
+          <div class="program__card--title"><h2>Workshop</h2></div>
+          <div class="program__card--text"><p>Try creating your own work using open source license rather than just watching.</p></div>
+        </div>
+        <div class="program__card flex">
+          <div class="program__card--icon"><img src="icons/program_icon_05.png" alt="Icon"></div>
+          <div class="program__card--title"><h2>Union</h2></div>
+          <div class="program__card--text"><p>Get opportunities to network with NE affiliates around the world, also after the conference.</p></div>
+        </div>
```

Comment:
- [ ]  While you make a great effort😊 I see that you forgot to use the article tags for this section, so it will be better if you use them as this from the requirements, so for this section💯😃, It contains a middle section with a column that includes a section header, a divider, five individual articles, each with an image, title, and description, and a line of text at the bottom.🚀👌

Reply:
@brainconnect93 Hello! 👋🏼 
Thanks for the feedback, but according to your advice I need to insert these cards into a section tag. But as you can see, these cards it's already in a section called 'program'.

Maybe I don't understand your advice. So I ask you to tell me again and explain better, please 🙌🏼